webapp/client/src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import LoginPage from './pages/LoginPage';
+
+const componentNameMap = {
+  'LoginPage': LoginPage
+}
+
+document.querySelectorAll('[data-is-component=yes]').forEach( (element) => {
```

Comment:
could we maybe name the function to make it easier understandable what is happening?
```suggestion
document.querySelectorAll('[data-is-component=yes]').forEach( function createReactComponents(element){
```

Reply:
Good idea, I'll pull it out. 👍